### ISSUE TITLE
Refresh Card View on view resize

### ISSUE THREAD

**Issue by mohit196** [JoniVR/VerticalCardSwiper#39]
### New Issue Checklist

- [x] I updated VerticalCardSwiper to the latest version.
- [x] I read the [Contribution Guidelines](CONTRIBUTING.md).
- [x] I read the [documentation](https://github.com/JoniVR/VerticalCardSwiper).
- [x] I searched for [existing GitHub issues](https://github.com/JoniVR/VerticalCardSwiper/issues).

### Issue Description

Right now If I am resizing the cardView the card cells are not adjusting to the new frame.

Steps: 
- Create a cardView Height constraint
- Assign a new value to height constraint. -- (Cell will not update its frame)

on reloading the CardView it is still giving me some random behavior because of the previous collection view height.

### Environment

- **iOS Version**: 12.0
- **Device(s)**: iPhone XR  Simulator
- **Xcode Version**: 10.2

**Comment by JoniVR** [comment]
> * Create a cardView Height constraint

Do you mean creating height constraints for objects inside the cardView? 
I would need some clarification on where exactly you are adding a constraint. 
A demo project would be even better.

Are you using the `sizeForItem` delegate function? 
Because that function is meant to allow you to change card sizes.

**Comment by mohit196** [comment]
I added an NSLayoutConstraint on cardSwiper view height and in didSwipeCardAway, I am updating the height constraint. After updating the cell should opt to new constraints. 

I have made some changes to the existing project. 

[VerticalCardSwiper-master-2.zip](https://github.com/JoniVR/VerticalCardSwiper/files/3032899/VerticalCardSwiper-master-2.zip)

**Comment by JoniVR** [comment]
So are you saying that the cards don't adapt their size right away once you swipe one away for the first time?

So does adding the following line:
```swift
cardSwiper.verticalCardSwiperView.layoutIfNeeded()
```
below the line where you update the constraint (kind of) fix it for you? (just so I understand what you're talking about)

**Comment by mohit196** [comment]
Thanks for update Joni

Use Case: 
- Swipe 2-3 cards upward.
- Now Swipe away the topmost card, you will see the issue.

![Simulator Screen Shot - iPhone XR - 2019-04-03 at 11 24 32](https://user-images.githubusercontent.com/22215577/55455883-12188c00-5603-11e9-82f5-f4fd7aca505e.png)

**Comment by JoniVR** [comment]
Yes, I see it now, alright I'll look into what's causing this sometime when I get the time for it. 
If you find a solution yourself in the meantime, feel free to post it here or submit a Pull Request.

**Comment by JoniVR** [comment]
Hi @mohit196 

Quick question, did you try my previous solution? 

If I call `cardSwiper.layoutIfNeeded()` after changing the constraint value it looks like it works to me?

```swift
func didSwipeCardAway(card: CardCell, index: Int, swipeDirection: SwipeDirection) {
    cardViewHeightConstraint.constant = 600.0
    cardViewHeightConstraint.priority = .required
    cardSwiper.layoutIfNeeded()
}
```

[VerticalCardSwiper-master-2.zip](https://github.com/JoniVR/VerticalCardSwiper/files/3068611/VerticalCardSwiper-master-2.zip)

**Comment by mohit196** [comment]
Yes, and the issue still exists. 

I tried:
cardSwiper.layoutIfNeeded()
and
cardSwiper.verticalCardSwiperView.layoutIfNeeded()
